docs/HowToDevelopThingsUsingSTThings.md

```diff
@@ -28,16 +28,16 @@ Then, follow the steps.
    ```  
 
 To develop App(SmartThings) UI and functionality, use the SmartThings SDK. Please refer to [SmartThings SDK](https://smartthings.developer.samsung.com/develop/workspace/general-tools/sdk.html) site.  
-The device definition is stored in a JSON configuration file that you need to provide [[Device Definition]](https://developer.tizen.org/development/iot-preview/iot-apis/things-sdk-api/device-definition)   
+The device definition is stored in a JSON configuration file that you need to provide [[Device Definition]](https://developer.tizen.org/development/iot-extension-sdk/api-guides/things-sdk-api/device-definition)
 
 ## Getting Started
-## 1. with Tizen Studio for RT
+### 1. with Tizen Studio for RT
```

Comment:
This is not sequence, it a choice among two options. Let's remove the numbering.

Reply:
Agreed, I will modify accordingly, thank you.